mindbender/maya/plugins/validate_project_editorial_info.py

```diff
@@ -0,0 +1,57 @@
+# validate_project_editorial_info.py
+import pyblish.api
+
+
+class ValidateMindbenderProjectEditInfo(pyblish.api.ContextPlugin):
+    """ Checks your scene with editorial info
```

Comment:
Because I didn't like how it looks inside of pyblish.
![screengrab_2](https://cloud.githubusercontent.com/assets/24475296/23813431/2cb3ddd8-05df-11e7-9875-ad58994ba807.gif)
But that might be because I am on a old laptop with low resolution screen, it might look diffrent on a high res desktop screen.